src/scenarios/shared/util.py

```diff
@@ -8,12 +8,6 @@
 from shared import const
 from performance.constants import UPLOAD_TOKEN_VAR
 
-def helixpayload():
```

Comment:
If you are going to remove this function from util.py then you will need to go through all of the places where we reference it from here and update them to use common.py instead. I would however recommend that you just instead keep the function definition here.